src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1575,7 +1575,8 @@ public void AssertAttachmentsFromReplication(DocumentsOperationContext context,
 
                         attachment.TryGet(nameof(AttachmentName.Name), out LazyStringValue attachmentName);
 
-                        var msg = $"Document '{id}' has attachment " +
+                        var type = doc.Flags.Contain(DocumentFlags.Revision) ? "Revision" : "Document";
+                        var msg = $"{type} '{doc.Id}' has attachment " +
```

Comment:
Does this message answers a Q: "What revision is problematic?" 

Shouldn't we identify the revision not only by document ID but also a CV?

Reply:
done